requirements.txt

```diff
@@ -4,3 +4,4 @@ google-api-python-client
 google-auth
 google-auth-httplib2
 google-auth-oauthlib
+google-cloud-bigquery==0.26.0
```

Comment:
not needed

Reply:
Should it not be pinned in requirements? See discussion here: https://github.com/pydata/pandas-gbq/pull/25#pullrequestreview-67723587